examples.html

```diff
@@ -9370,6 +9371,70 @@ <h4 class="subtitle is-4">C4 PlantUML</h4>
             </svg>
         </div>
 
+        <h3 id="structurizr-container" class="title is-3">Container diagram<a class="anchor-link" href="#structurizr-container">#</a></h3>
+        <h4 class="subtitle is-4">Structurizr</h4>
+        <div class="content">
+            <pre class="diagram-source"><code class="language-structurizr">
+workspace {
+    model { 
+        user = person "User" 
+        softwareSystem = softwareSystem "Software System" { 
+            webapp = container "Web Application" { 
+                user -> this "Uses!!!" 
+            } 
+            database = container "Database" { 
+                webapp -> this "Reads from and writes to" 
+            } 
+        } 
+    } 
+    views { 
+        systemContext softwareSystem { 
+            include * 
+            autolayout lr 
+        } 
+        container softwareSystem { 
+            include * 
+            autolayout lr 
+        } 
+        theme default 
+    } 
+}
+</code></pre>
+            <div class="highlight">
+                <pre><code class="language-http">GET https://kroki.io/structurizr/svg/eNq1UEFOxDAMvPMKN0ckngASghewQpzdxtVGpHEUOxtWq_07aam62y4cmUsyEzszNhROnxKxIzjdQcXAljycYCIjslCCR4iUhAOY90rN5VW414KJdkdRGmrdRjC7mcOPYK6_HlGoxRhrY8dB0YVqZj6ohecYvetQHYebniXWwxPo3smUSpqmMeu685paVGxRaG32Oqu_uszpFp83QivQJx4Ag4WSnJKA8t_G83U-Do6KXBvJtJWXGoe-dLu8TSAXOp8twf1axqzs8chZwacb4xGXaf_JQPc0EFjqMXtdBj5_A1fGoBA=</code></pre>
```

Comment:
you can add `?view-key=SoftwareSystem-Container` (otherwise we get a random view)

Reply:
Added view-key (changed to other diagram examples, so view-key is different)